app/javascript/src/components/Notes/Note.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Dropdown, Typography, Tag, Avatar, Tooltip } from "neetoui";
+import { getTimeFromNow, getDayAndTimeFromDate } from "utils";
+
+const Note = ({ title, description, createdAt, noteType }) => {
+  const { Menu, MenuItem } = Dropdown;
+
+  return (
+    <div className="mx-4 my-2 flex flex-col border border-solid p-5 shadow">
+      <div className="flex flex-row justify-between">
+        <Typography style="h4">{title}</Typography>
+        <Dropdown className="bg-transparent" icon={MenuVertical}>
+          <Menu>
+            <MenuItem.Button>Edit</MenuItem.Button>
+            <MenuItem.Button>Delete</MenuItem.Button>
```

Comment:
Go through https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663060378971029 very carefully and fix the issue over here. Moving forward, I expect you to not repeat the same mistakes mentioned in that Slack thread.

Also go through https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663754719557349

Reply:
I found two issues here based on the thread:
- Line 14: Don't override with custom colours using className, use component provided styles or neeto-ui custom CSS properties.
- Line 17: Provide a danger style to the Delete button.

Is my understanding correct?

Reply:
Correct. 👍